### PR TITLE
Daily economic indicators based on Google searches

### PR DESCRIPTION
**Pull request by christophsax**
Thanks for contributing your submission to the e-Rum2020 CovidR contest.

**For CovidR contributors:**

- [X] The title of this PR reflects your contribution
- [X] The pull request only includes one new `.yml` file under `contest/contributions`
- [x] Make sure "Allow edits from maintainers" is checked
- [x] Send an email to covidrcontest@erum.io (for further direct communication down the road)
     - Subject: "Submission to CovidR: PR #00", where #00 is the number of this Pull Request
     - Content: A full link to this Pull Request
     - Optionally include your social media profiles (Twitter, LinkedIn, Facebook): we will use them to `@mention` you while sharing the contribution through our channels.

---

For CovidR maintainers:

- [x] Label the PR as "submission"
- [x] Check the validity of the submission
- [x] Assess completeness of metadata and possibly adjust the `.yml` file-name
- [x] Check the integration of the new page in the website gallery (via Actions' artifact or locally)
- [x] Make sure the email to covidrcontest@erum.io was received
- [x] Add a field with the current `date` and approx. merge time (e.g. `2020-04-01-16:30` for April 1st, 4:30 PM)
- [x] Squash-merge the pull request (commit headline will be OK, consider cleaning-up the details)
- [x] The initial comment should be automatically created, otherwise comment manually with the raw content of [vote-comment.md](../tree/master/contest/_tools/vote-comment.md) and an initial like
- [x] Provide here the link to the contribution as published to the gallery: https://milano-r.github.io/erum2020-covidr-contest/trendecon.html
- [ ] Instruct contributors on how to include their badge from the gallery page, share on social media, and get a [free ticket](https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686).
- [x] Consolidate e-mailed information.

### CODE REVIEW

File: contest/contributions/trendecon.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Contribution's title, author, abstract
+title: Daily economic indicators based on Google searches # title case
+author: Angelica Becerra, Vera Z. Eichenauer, Ronald Indergand, Stefan Legge,
+  Isabel Martinez, Nina Mühlebach, Furkan Oguz, Christoph Sax,
+  Kristina Schuepbach, Severin Thöni, Uwe Thümmel
```
Comment (riccardoporreca) [inline contest/contributions/trendecon.yml]: We would typically prefer up to 4 names listed as `author`, possibly using et al., for a better integration in the [CovidR gallery](https://milano-r.github.io/erum2020-covidr-contest/). This is not strictly required though, I understand you are trying to be fair to the many contributors.

File: contest/contributions/trendecon.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  An accompaning R package contains the code to construct long daily time series
+  from Google Trends. Robustness of the data is achieved by querying Google
+  mulitple times. The queries are sampled at daily, weekly and monthy
+  frequencies and then harmonized such that the long term trend is preserved. A
+  more detailed methodological description is given on the
+  [website](https://www.trendecon.org/#method). We are currently summarizing
+  these results in a research paper.
+# Short name to be used in the website Gallery menu (can be the same as title)
+menu_entry: Daily economic indicators based on Google searches
```
Comment (riccardoporreca) [inline contest/contributions/trendecon.yml]: Shorter `menu_entry` better fit to the Gallery menu
```suggestion
menu_entry: Daily economic indicators
```

File: contest/contributions/trendecon.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Contribution's title, author, abstract
+title: Daily economic indicators based on Google searches # title case
```
Comment (riccardoporreca) [inline contest/contributions/trendecon.yml]: Title case
```suggestion
title: Daily Economic Indicators Based on Google Searches
```

**Review by riccardoporreca — CHANGES_REQUESTED** [review]
@christophsax, Thanks for your submission, see bellow some minor "cosmetic" comments.

**Comment by riccardoporreca** [comment]
@christophsax, we haven't found any email sent to covidrcontest@erum.io, can you please check?
> - [x] Send an email to covidrcontest@erum.io (for further direct communication down the road)
     - Subject: "Submission to CovidR: PR #00", where #00 is the number of this Pull Request
     - Content: A full link to this Pull Request
     - Optionally include your social media profiles (Twitter, LinkedIn, Facebook): we will use them to `@mention` you while sharing the contribution through our channels.

File: contest/contributions/trendecon.yml (hunk 1 above)
Comment (christophsax) [reply contest/contributions/trendecon.yml]: Perhaps: 'trendecon team'? We have the contributors on the gist.

**Comment by christophsax** [comment]
Thanks, Riccardo!

I did send it, but perhaps it got lost somewhere on its way.

image

File: contest/contributions/trendecon.yml (hunk 1 above)
Comment (riccardoporreca) [reply contest/contributions/trendecon.yml]: Sounds like a good choice, I recommend to include the name(s) of the person potentially presenting the project at the pre-conference event, you can use stg like
```yaml
authors: Angelica Becerra et al., trendecon team
```

**Comment by riccardoporreca** [comment]
> I did send it, but perhaps it got lost somewhere on its way.

I guess so: don't worry, I will fetch the contact information from the screenshot above.

**Comment by christophsax** [comment]
So you don't have to use OCR...

Submitted our project on Daily economic indicators based on Google searches to CovidR 

https://github.com/Milano-R/erum2020-covidr-contest/pull/94

Thanks!

Christoph


Twitter:

@angelicamabg
@IZMartinez86
@vz_eich
@christoph_sax

**Review by riccardoporreca — APPROVED** [review]
There are currently issues with GitHub preventing the integration of contributions (https://www.githubstatus.com/).
We will finalize submissions once such issues are solved.

File: contest/contributions/trendecon.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  frequencies and then harmonized such that the long term trend is preserved. A
+  more detailed methodological description is given on the
+  [website](https://www.trendecon.org/#method). We are currently summarizing
+  these results in a research paper.
+# Short name to be used in the website Gallery menu (can be the same as title)
+menu_entry: Daily economic indicators
+# URL of the public source code repository
+repository: https://github.com/trendecon
+type: gist
+content: <script src="https://gist.github.com/angelicambg/06307a697c54f09b88ecfe602873c0cd.js"></script> # place here what you get from the "Embed" button
```
Comment (riccardoporreca) [inline contest/contributions/trendecon.yml:25]:
```suggestion
content: <script src="https://gist.github.com/angelicambg/06307a697c54f09b88ecfe602873c0cd.js"></script> # place here what you get from the "Embed" button
date: 2020-05-22-23:59
```

**Review by riccardoporreca — APPROVED** [review]

**Comment by riccardoporreca** [comment]
Integration of contributions in the CovidR Gallery still blocked by an ongoing GitHub incident

**Comment by riccardoporreca** [comment]
@christophsax, the major GitHub incident was solved and now contributions are fully integrated.

Your contribution is now accepted and visible at https://milano-r.github.io/erum2020-covidr-contest/trendecon.html :tada: 
You can click on the **eRum2020::Covidr badge** in the page to obtain the badge code you can embed in your repository.

Furthermore, feel free to spread your contribution on social media with the link above, using hashtags `#erum2020` `#covidr`, and invite the community to like the contribution using the corresponding button.

Last but not least: You can get your **free ticket for the pre-conference event** at https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686

**Pull request merged**